types-fixup.mjs

```diff
@@ -49,7 +49,8 @@ fs.writeFileSync(path, dts);
 
 const animationComponentProps = [
     ['activate', 'boolean'],
-    ['assets', 'any[]'],
+    ['assets', 'Asset[]|number[]'],
+    ['animations', '{[string]: Animation}'],
```

Comment:
This line doesn't, I end up with two definitions of the 'animations' property in the types file.